cvat-ui/src/components/annotation-page/canvas/views/canvas2d/canvas-wrapper.tsx

```diff
@@ -146,13 +149,16 @@ interface DispatchToProps {
     onGetDataFailed(error: Error): void;
     onCanvasErrorOccurred(error: Error): void;
     onStartIssue(position: number[]): void;
+    onUpdateEditedObject(objectType: ShapeType | null, editedState: ObjectState | null): void;
```

Comment:
It seems, the first argument can't be null

```suggestion
    onUpdateEditedObject(objectType: ShapeType, editedState: ObjectState | null): void;
```

Reply:
At least now you do not call it with `null` here.